### PR TITLE
Update front matter, acknowledgements and criticality for EXO

### PR DESCRIPTION
**Pull request by ahuynhMITRE**
## 🗣 Description ##

Updating the front-matter and acknowledgements to the EXO SCB to match language approved by the CISA Front office. Also added the html formatting for criticality to support new field in report. 

## 💭 Motivation and context ##

Changes are required to ensure consistency across SCBs for the approved front-matter and acknowledgements. Criticality HTML is added to support new report field and supporting automation.  

#471 
#341 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: baselines/exo.md [hunk 1]
```diff
@@ -112,12 +113,14 @@ documentation provides some helpful starting points.
...
 A list of approved IP addresses for sending mail SHALL be maintained.
 
+<!--Policy: MS.EXO.2.1v1; Criticality: SHALL -->
 - _Rationale:_ Failing to maintain an accurate list of authorized IP addresses may result in spoofed email messages or failure to deliver legitimate messages when SPF is enabled.  Maintaining such a list ensures that unauthorized servers sending spoofed messages can be detected and permit messages from legitimate senders to be delivered.
 - _Last modified:_ June 2023
 
 #### MS.EXO.2.2v1
 An SPF policy(s) that designates only these addresses as approved senders SHALL be published.
 
+<!--Policy: MS.EXO.2.2v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:123]: Rego has 'Should/Not-Implemented'.  We need to either correct Rego, Secure Baseline document or both.  In this case I assume both should be "Shall/Not-Implemented".

File: baselines/exo.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ forwarding rules to exfiltrate data to external recipients.
 #### MS.EXO.1.1v1
 Automatic forwarding to external domains SHALL be disabled.
 
+<!--Policy: MS.EXO.1.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:64]: Not super important but we are not using all uppercase for criticallity.  I kind of like the all caps.

File: baselines/exo.md [hunk 3]
```diff
@@ -188,6 +191,7 @@ custom domains. Exchange Online Protection (EOP) features include DKIM signing c
 #### MS.EXO.3.1v1
 DKIM SHOULD be enabled for any custom domain.
 
+<!--Policy: MS.EXO.3.1v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:194]: Rego has Shall.  Open Issue?

File: baselines/exo.md
Comment (crutchfield) [inline baselines/exo.md:1]: Rego has 3.2v1 and 3.3v1 that are not here.

File: baselines/exo.md [hunk 4]
```diff
@@ -112,12 +113,14 @@ documentation provides some helpful starting points.
 #### MS.EXO.2.1v1
 A list of approved IP addresses for sending mail SHALL be maintained.
 
+<!--Policy: MS.EXO.2.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:116]:
```suggestion
<!--Policy: MS.EXO.2.1v1; Criticality: SHALL/NOT-IMPLEMENTED -->
```

File: baselines/exo.md [hunk 5]
```diff
@@ -397,6 +406,7 @@ the following policies.
 #### MS.EXO.6.1v1
 Contact folders SHALL NOT be shared with all domains.
 
+<!--Policy: MS.EXO.6.1v1; Criticality: SHALL NOT -->
```
Comment (crutchfield) [inline baselines/exo.md:409]: I expected just SHALL like we have in Rego.

Comment (crutchfield) [inline baselines/exo.md:409]:
```suggestion
<!--Policy: MS.EXO.6.1v1; Criticality: SHALL -->
```

File: baselines/exo.md [hunk 6]
```diff
@@ -405,6 +415,7 @@ for specific legitimate uses as needed.
 #### MS.EXO.6.2v1
 Calendar details SHALL NOT be shared with all domains.
 
+<!--Policy: MS.EXO.6.2v1; Criticality: SHALL NOT -->
```
Comment (crutchfield) [inline baselines/exo.md:418]:
```suggestion
<!--Policy: MS.EXO.6.2v1; Criticality: SHALL -->
```

File: baselines/exo.md [hunk 7]
```diff
@@ -551,12 +563,14 @@ required, guidance for configuring Microsoft’s DLP solution can be found in fo
 #### MS.EXO.8.1v1
 A DLP solution SHALL be used. The selected DLP solution SHOULD offer services comparable to the native DLP solution offered by Microsoft.
 
+<!--Policy: MS.EXO.8.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:566]:
```suggestion
<!--Policy: MS.EXO.8.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md (hunk 2 above)
Comment (ahuynhMITRE) [reply baselines/exo.md:64]: I would keep criticality as is to match the other headers in the report are structured.

File: baselines/exo.md [hunk 8]
```diff
@@ -551,12 +563,14 @@ required, guidance for configuring Microsoft’s DLP solution can be found in fo
...
 - _Rationale:_ Users may inadvertently disclose sensitive information to unauthorized individuals. A capable DLP solution should detect the presence of sensitive information in Exchange Online and block access to authorized entities.
 - _Last modified:_ June 2023
 
 #### MS.EXO.8.2v1
 The DLP solution SHALL protect PII and sensitive information, as defined by the agency. At a minimum, the sharing of credit card numbers, Taxpayer Identification Numbers (TIN), and Social Security Numbers (SSN) via email SHALL be restricted.
 
+<!--Policy: MS.EXO.8.2v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:573]:
```suggestion
<!--Policy: MS.EXO.8.2v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 9]
```diff
@@ -597,6 +611,7 @@ Microsoft Defender can be found in the follow section of the CISA M365 Security
 #### MS.EXO.9.1v1
 Emails SHALL be filtered by the file types of included attachments. The selected filtering solution SHOULD offer services comparable to Microsoft Defender's Common Attachment Filter.
 
+<!--Policy: MS.EXO.9.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:614]:
```suggestion
<!--Policy: MS.EXO.9.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 10]
```diff
@@ -607,6 +622,7 @@ prevent the spread of malware distributed via click-to-run email attachments.
 #### MS.EXO.9.2v1
 The attachment filter SHOULD attempt to determine the true file type and assess the file extension.
 
+<!--Policy: MS.EXO.9.2v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:625]:
```suggestion
<!--Policy: MS.EXO.9.2v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 11]
```diff
@@ -616,6 +632,7 @@ file extension can help detect instances where the file extension has been chang
 #### MS.EXO.9.3v1
 Disallowed file types SHALL be determined and set. At a minimum, click-to-run files SHOULD be blocked (e.g., .exe, .cmd, and .vbe).
 
+<!--Policy: MS.EXO.9.3v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:635]:
```suggestion
<!--Policy: MS.EXO.9.3v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 12]
```diff
@@ -670,6 +687,7 @@ implement malware scanning, see the following policies of the CISA M365 Security
 #### MS.EXO.10.1v1
 Emails SHALL be scanned for malware.
 
+<!--Policy: MS.EXO.10.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:690]:
```suggestion
<!--Policy: MS.EXO.10.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 13]
```diff
@@ -678,6 +696,7 @@ the risk for end users.
 #### MS.EXO.10.2v1
 Emails identified as containing malware SHALL be quarantined or dropped.
 
+<!--Policy: MS.EXO.10.2v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:699]:
```suggestion
<!--Policy: MS.EXO.10.2v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 14]
```diff
@@ -686,6 +705,7 @@ users cannot interact with those emails.
 #### MS.EXO.10.3v1
 Email scanning SHALL be capable of reviewing emails after delivery.
 
+<!--Policy: MS.EXO.10.3v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:708]:
```suggestion
<!--Policy: MS.EXO.10.3v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 15]
```diff
@@ -757,6 +777,7 @@ Defender for Office 365. If using Defender for Office 365 for impersonation prot
 #### MS.EXO.11.1v1
 Impersonation protection checks SHOULD be used.
 
+<!--Policy: MS.EXO.11.1v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:780]:
```suggestion
<!--Policy: MS.EXO.11.1v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 16]
```diff
@@ -766,6 +787,7 @@ senders, the risk of a successful phishing attempt can be reduced.
 #### MS.EXO.11.2v1
 User warnings, comparable to the user safety tips included with EOP, SHOULD be displayed.
 
+<!--Policy: MS.EXO.11.2v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:790]:
```suggestion
<!--Policy: MS.EXO.11.2v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 17]
```diff
@@ -775,6 +797,7 @@ successful phishing attempts.
 #### MS.EXO.11.3v1
 The phishing protection solution SHOULD include an AI-based phishing detection tool comparable to EOP Mailbox Intelligence.
 
+<!--Policy: MS.EXO.11.3v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:800]:
```suggestion
<!--Policy: MS.EXO.11.3v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 18]
```diff
@@ -836,13 +859,15 @@ security mechanisms.
 #### MS.EXO.12.1v1
 IP allow lists SHOULD NOT be created.
 
+<!--Policy: MS.EXO.12.1v1; Criticality: SHOULD NOT -->
```
Comment (crutchfield) [inline baselines/exo.md:862]:
```suggestion
<!--Policy: MS.EXO.12.1v1; Criticality: SHOULD -->
```

File: baselines/exo.md [hunk 19]
```diff
@@ -836,13 +859,15 @@ security mechanisms.
...
 - _Rationale:_ Messages sent from IP addresses on an allow list bypass important
 security mechanisms, including spam filtering and sender authentication checks.  Avoiding use of IP allow lists prevents potential threats from circumventing security mechanisms.
 - _Last modified:_ June 2023
 
 #### MS.EXO.12.2v1
 Safe lists SHOULD NOT be enabled.
 
+<!--Policy: MS.EXO.12.2v1; Criticality: SHOULD NOT -->
```
Comment (crutchfield) [inline baselines/exo.md:870]:
```suggestion
<!--Policy: MS.EXO.12.2v1; Criticality: SHOULD -->
```

File: baselines/exo.md [hunk 20]
```diff
@@ -988,20 +1014,23 @@ the CISA M365 Security Configuration Baseline for Defender for Office 365.
 #### MS.EXO.14.1v1
 A spam filter SHALL be enabled. The filtering solution selected SHOULD offer services comparable to the native spam filtering offered by Microsoft.
 
+<!--Policy: MS.EXO.14.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:1017]:
```suggestion
<!--Policy: MS.EXO.14.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 21]
```diff
@@ -988,20 +1014,23 @@ the CISA M365 Security Configuration Baseline for Defender for Office 365.
...
 
+<!--Policy: MS.EXO.14.2v1; Criticality: SHALL -->
 - _Rationale:_ Spam is a constant threat as junk mail can reduce user productivity, fill up mailboxes unnecessarily, and in some cases include malicious links or attachments.
 Moving spam messages to a separate junk or quarantine folder helps users filter out spam while still giving them the ability to review messages, as needed, in case a message is filtered incorrectly.
 - _Last modified:_ June 2023
 
 #### MS.EXO.14.3v1
 Allowed domains SHALL NOT be added to inbound anti-spam protection policies.
 
+<!--Policy: MS.EXO.14.3v1; Criticality: SHALL NOT -->
```
Comment (crutchfield) [inline baselines/exo.md:1033]:
```suggestion
<!--Policy: MS.EXO.14.3v1; Criticality: SHALL/3RD PARTY-->
```

File: baselines/exo.md [hunk 22]
```diff
@@ -1079,12 +1108,14 @@ If the agency is using Microsoft Defender for Office 365 to meet this baseline p
 #### MS.EXO.15.1v1
 URL comparison with a block-list SHOULD be enabled.
 
+<!--Policy: MS.EXO.15.1v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:1111]:
```suggestion
<!--Policy: MS.EXO.15.1v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 23]
```diff
@@ -1079,12 +1108,14 @@ If the agency is using Microsoft Defender for Office 365 to meet this baseline p
...
 - _Rationale:_ Users may be directed to malicious websites via links in email. Blocking access to known, malicious URLs can prevent users from accessing known malicious websites.
 - _Last modified:_ June 2023
 
 #### MS.EXO.15.2v1
 Direct download links SHOULD be scanned for malware.
 
+<!--Policy: MS.EXO.15.2v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:1118]:
```suggestion
<!--Policy: MS.EXO.15.2v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 24]
```diff
@@ -1093,6 +1124,7 @@ users from infecting their devices.
 #### MS.EXO.15.3v1
 User click tracking SHOULD be enabled.
 
+<!--Policy: MS.EXO.15.3v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:1127]:
```suggestion
<!--Policy: MS.EXO.15.3v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 25]
```diff
@@ -1168,6 +1200,7 @@ At a minimum, the following alerts SHALL be enabled:
 
   g. **A potentially malicious URL click was detected.**
 
+<!--Policy: MS.EXO.16.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:1203]:
```suggestion
<!--Policy: MS.EXO.16.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 26]
```diff
@@ -1177,6 +1210,7 @@ At a minimum, the following alerts SHALL be enabled:
 #### MS.EXO.16.2v1
 The alerts SHOULD be sent to a monitored address or incorporated into a SIEM.
 
+<!--Policy: MS.EXO.16.2v1; Criticality: SHOULD -->
```
Comment (crutchfield) [inline baselines/exo.md:1213]:
```suggestion
<!--Policy: MS.EXO.16.2v1; Criticality: SHOULD/3RD PARTY -->
```

File: baselines/exo.md [hunk 27]
```diff
@@ -1240,6 +1274,7 @@ Office 365.
 #### MS.EXO.17.1v1
 Microsoft Purview Audit (Standard) logging SHALL be enabled.
 
+<!--Policy: MS.EXO.17.1v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:1277]:
```suggestion
<!--Policy: MS.EXO.17.1v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 28]
```diff
@@ -1251,6 +1286,7 @@ name, Unified Audit Logs).
 #### MS.EXO.17.2v1
 Microsoft Purview Audit (Premium) logging SHALL be enabled.
 
+<!--Policy: MS.EXO.17.2v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:1289]:
```suggestion
<!--Policy: MS.EXO.17.2v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md [hunk 29]
```diff
@@ -1259,6 +1295,8 @@ Furthermore, it is required for government agencies by OMB M-21-13 (referred to
 
 #### MS.EXO.17.3v1
 Audit logs SHALL be maintained for at least the minimum duration dictated by [OMB M-21-31 (Appendix C)](https://www.whitehouse.gov/wp-content/uploads/2021/08/M-21-31-Improving-the-Federal-Governments-Investigative-and-Remediation-Capabilities-Related-to-Cybersecurity-Incidents.pdf).
+
+<!--Policy: MS.EXO.17.3v1; Criticality: SHALL -->
```
Comment (crutchfield) [inline baselines/exo.md:1299]:
```suggestion
<!--Policy: MS.EXO.17.3v1; Criticality: SHALL/3RD PARTY -->
```

File: baselines/exo.md (hunk 19 above)
Comment (crutchfield) [reply baselines/exo.md:870]: I resolved all the NOT-IMPLEMENTED/3RD PARTY.  We decided to drop NOT, right?

**Review by crutchfield — APPROVED** [review]
Look at unresolved comment on MS.EXO.12.2v1.  Looks good. Approving.

**Comment by ahuynhMITRE** [comment]
this is ready to merge @nanda-katikaneni

**Pull request merged**